docs/topics/http/urls.txt

```diff
@@ -368,6 +368,24 @@ the following example is valid::
 In the above example, the captured ``"username"`` variable is passed to the
 included URLconf, as expected.
 
+Nested captured parameters are discarded by Django when urls are reversed,
```

Comment:
URLs
Could you add an example for this paragraph? It's not easy for me to understand what you mean just by reading the description.